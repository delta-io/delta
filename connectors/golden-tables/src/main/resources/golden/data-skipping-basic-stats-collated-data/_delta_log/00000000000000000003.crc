{"txnId":"8819068d-0d79-4102-9cf2-925d17a06951","tableSizeBytes":2126,"numFiles":3,"numDeletedRecordsOpt":0,"numDeletionVectorsOpt":0,"numMetadata":1,"numProtocol":1,"setTransactions":[],"domainMetadata":[],"metadata":{"id":"03e91258-268e-49d5-ad70-3cb02d29c1e7","format":{"provider":"parquet","options":{}},"schemaString":"{\"type\":\"struct\",\"fields\":[{\"name\":\"c1\",\"type\":\"string\",\"nullable\":true,\"metadata\":{\"__COLLATIONS\":{\"c1\":\"icu.UNICODE_CI\"}}},{\"name\":\"c2\",\"type\":\"string\",\"nullable\":true,\"metadata\":{}}]}","partitionColumns":[],"configuration":{"delta.enableDeletionVectors":"true"},"createdTime":1727111843058},"protocol":{"minReaderVersion":3,"minWriterVersion":7,"readerFeatures":["deletionVectors"],"writerFeatures":["deletionVectors","collations-dev","appendOnly","invariants"]},"histogramOpt":{"sortedBinBoundaries":[0,8192,16384,32768,65536,131072,262144,524288,1048576,2097152,4194304,8388608,12582912,16777216,20971520,25165824,29360128,33554432,37748736,41943040,50331648,58720256,67108864,75497472,83886080,92274688,100663296,109051904,117440512,125829120,130023424,134217728,138412032,142606336,146800640,150994944,167772160,184549376,201326592,218103808,234881024,251658240,268435456,285212672,301989888,318767104,335544320,352321536,369098752,385875968,402653184,419430400,436207616,452984832,469762048,486539264,503316480,520093696,536870912,553648128,570425344,587202560,603979776,671088640,738197504,805306368,872415232,939524096,1006632960,1073741824,1140850688,1207959552,1275068416,1342177280,1409286144,1476395008,1610612736,1744830464,1879048192,2013265920,2147483648,2415919104,2684354560,2952790016,3221225472,3489660928,3758096384,4026531840,4294967296,8589934592,17179869184,34359738368,68719476736,137438953472,274877906944],"fileCounts":[3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"totalBytes":[2126,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]},"deletedRecordCountsHistogramOpt":{"deletedRecordCounts":[3,0,0,0,0,0,0,0,0,0]},"allFiles":[{"path":"test%25file%25prefix-part-00000-0c9f0635-2f05-4a95-8e77-917d2ab27302-c000.snappy.parquet","partitionValues":{},"size":710,"modificationTime":1727111847322,"dataChange":false,"stats":"{\"numRecords\":3,\"minValues\":{\"c1\":\"a\",\"c2\":\"1\"},\"maxValues\":{\"c1\":\"c\",\"c2\":\"3\"},\"nullCount\":{\"c1\":0,\"c2\":0},\"tightBounds\":true,\"statsWithCollation\":{\"icu.UNICODE_CI.153.120.0.0\":{\"minValues\":{\"c1\":\"a\"},\"maxValues\":{\"c1\":\"c\"}}}}","tags":{"INSERTION_TIME":"1727111847322000","MIN_INSERTION_TIME":"1727111847322000","MAX_INSERTION_TIME":"1727111847322000","OPTIMIZE_TARGET_SIZE":"268435456"}},{"path":"test%25file%25prefix-part-00000-6ae7a50e-b5ee-471c-bc3c-4ebac9e44dc3-c000.snappy.parquet","partitionValues":{},"size":706,"modificationTime":1727111849016,"dataChange":false,"stats":"{\"numRecords\":3,\"minValues\":{\"c1\":\"C\",\"c2\":\"4\"},\"maxValues\":{\"c1\":\"C\",\"c2\":\"6\"},\"nullCount\":{\"c1\":2,\"c2\":0},\"tightBounds\":true,\"statsWithCollation\":{\"icu.UNICODE_CI.153.120.0.0\":{\"minValues\":{\"c1\":\"C\"},\"maxValues\":{\"c1\":\"C\"}}}}","tags":{"INSERTION_TIME":"1727111849016000","MIN_INSERTION_TIME":"1727111849016000","MAX_INSERTION_TIME":"1727111849016000","OPTIMIZE_TARGET_SIZE":"268435456"}},{"path":"test%25file%25prefix-part-00000-b14da47c-42b8-47ad-88c2-4e268d1bb03f-c000.snappy.parquet","partitionValues":{},"size":710,"modificationTime":1727111848383,"dataChange":false,"stats":"{\"numRecords\":3,\"minValues\":{\"c1\":\"A\",\"c2\":\"4\"},\"maxValues\":{\"c1\":\"C\",\"c2\":\"6\"},\"nullCount\":{\"c1\":0,\"c2\":0},\"tightBounds\":true,\"statsWithCollation\":{\"icu.UNICODE_CI.153.120.0.0\":{\"minValues\":{\"c1\":\"A\"},\"maxValues\":{\"c1\":\"C\"}}}}","tags":{"INSERTION_TIME":"1727111848383000","MIN_INSERTION_TIME":"1727111848383000","MAX_INSERTION_TIME":"1727111848383000","OPTIMIZE_TARGET_SIZE":"268435456"}}]}
