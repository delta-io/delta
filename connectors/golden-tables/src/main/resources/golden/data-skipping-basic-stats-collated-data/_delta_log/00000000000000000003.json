{"commitInfo":{"timestamp":1726521401673,"operation":"WRITE","operationParameters":{"mode":"Append","statsOnLoad":false,"partitionBy":"[]"},"readVersion":2,"isolationLevel":"WriteSerializable","isBlindAppend":true,"operationMetrics":{"numFiles":"1","numOutputRows":"3","numOutputBytes":"475"},"tags":{"noRowsCopied":"true","restoresDeletedRows":"false"},"engineInfo":"Databricks-Runtime/<unknown>","txnId":"4d6b94f9-d743-4b08-898a-e007b05b33fa"}}
{"add":{"path":"test%25file%25prefix-part-00000-b5cb0092-c62b-4d11-80ce-00aad14a801e-c000.snappy.parquet","partitionValues":{},"size":475,"modificationTime":1726521401605,"dataChange":true,"stats":"{\"numRecords\":3,\"minValues\":{},\"maxValues\":{},\"nullCount\":{\"c1\":3},\"tightBounds\":true,\"statsWithCollation\":{\"icu.UNICODE_CI.153.120.0.0\":{\"minValues\":{},\"maxValues\":{}}}}","tags":{"INSERTION_TIME":"1726521401605000","MIN_INSERTION_TIME":"1726521401605000","MAX_INSERTION_TIME":"1726521401605000","OPTIMIZE_TARGET_SIZE":"268435456"}}}
